spec/site/command_spec.rb

```diff
@@ -211,7 +211,37 @@
   it 'Test time synchronization', sxl: '>=1.0.7' do |example|
     TestSite.connected do |task,supervisor,site|
       prepare task, site
-      verify_synchronization
+      @site.log "Set date", level: :test
+      command_list = build_command_list :M0104, :setDate, {
+        securityCode: SECRETS['security_codes'][1],
+        year: "2020",
+        month: "9",
+        day: "29",
+        hour: "17",
+        minute: "29",
+        second: "51"
+      }
+      send_command_and_confirm @task, command_list, "intention to set date"
+      status_list = { S0096: [
+          :year,
+          :month,
+          :day,
+          :hour,
+          :minute,
+          :second,
+        ] }
+      message, result = @site.request_status @component, convert_status_list(status_list), collect: {
+        timeout: SUPERVISOR_CONFIG['status_update_timeout']
+      }
+      status = "S0096"
+      expect(result[{"sCI" => status, "n" => "year"}]["s"]).to be == "2020"
+      expect(result[{"sCI" => status, "n" => "month"}]["s"]).to be == "9"
+      expect(result[{"sCI" => status, "n" => "day"}]["s"]).to be == "29"
+      expect(result[{"sCI" => status, "n" => "hour"}]["s"]).to be == "17"
+      expect(result[{"sCI" => status, "n" => "minute"}]["s"]).to be == "29"
+      expect(result[{"sCI" => status, "n" => "second"}]["s"]).to be == "51"
```

Comment:
i don't think you can expect the second or minute to the the same as what you set, since the imte passes between setting and getting the clock

Reply:
That makes sense, I didn't consider that time passes.

Reply:
Maybe a more robust approach would be to simply diff the requested and the received time, and expect them to be within some configured range. That would verify the content of the minutes and seconds atttributes, which don't look at now, while taking into account network delays.

Something along the lines of:
```ruby
time_diff = (received - set).abs
expect( time_diff ).to be <= max_diff
```